### ISSUE TITLE
[Search v1] Implement ApproveMoneyRequestOnSearch command

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#46073]
Implement [this section](https://docs.google.com/document/d/1ldZ85fyQO4662hLZRNFfm0_IxaP7-PWvaqc8Z33jsLQ/edit#bookmark=id.kt71e0n7yfsm) of the doc for the Approve command only

**Comment by luacmartins** [comment]
No updates yet. I'll start working on it this week.

**Comment by johncschuster** [comment]
Bumping to keep Melvin at bay

**Comment by johncschuster** [comment]
Bumping to keep Melvin at bay

**Comment by johncschuster** [comment]
Another bump for Melv

**Comment by johncschuster** [comment]
Bumping for Melv.

@luacmartins, should we keep this at `Daily`?

**Comment by johncschuster** [comment]
@luacmartins are we still blocked on something?

**Comment by luacmartins** [comment]
Yes, Auth and Next steps being moved to Auth

**Comment by johncschuster** [comment]
Bumping to keep Melvin at bay 👍

**Comment by luacmartins** [comment]
On hold

**Comment by johncschuster** [comment]
Still on hold

**Comment by johncschuster** [comment]
Still on hold

**Comment by johncschuster** [comment]
Still on hold

**Comment by johncschuster** [comment]
Still on hold

**Comment by johncschuster** [comment]
Oooooh, it looks like we're off hold! 🙌

**Comment by johncschuster** [comment]
bump

**Comment by lakchote** [comment]
Yes! Auth [PR](https://github.com/Expensify/Auth/pull/12914) is under review.

**Comment by luacmartins** [comment]
We can close this issue. I'll handle hooking the command up in App as part of https://github.com/Expensify/App/issues/51898
